bottlecap/src/tags/lambda/tags.rs

```diff
@@ -105,6 +105,16 @@ fn tags_from_env(
     if let Ok(memory_size) = std::env::var(MEMORY_SIZE_VAR) {
         tags_map.insert(MEMORY_SIZE_KEY.to_string(), memory_size);
     }
+
+    if let Some(tags) = &config.tags {
+        for tag in tags.split(',') {
```

Comment:
This looks like it might fail if someone adds a tag like `a:b,c:b:d` ignoring the second one, should we warn or do something when this happens?

Reply:
Actually that's completely allowed and we do this intentionally in the version tag. We treat `b:d` as one string.

You can see it in the main agent: https://github.com/DataDog/datadog-agent/blob/main/pkg/serverless/tags/tags.go#L120

which is faithfully replicated here in lambda tags.

Reply:
Thanks!